academic-services/graph/network-similarity-databricks-getsimilarity.md

```diff
@@ -0,0 +1,42 @@
+---
+title: NetworkSimilarity.getSimilarity Function
+description: NetworkSimilarity.getSimilarity Function
+services: microsoft-academic-services
+ms.topic: extra
+ms.service: microsoft-academic-services
+ms.date: 9/23/2020
+---
+# NetworkSimilarity.getSimilarity Function
+
+### Compute similarity score between two entities.
+
+## float getSimilarity(e1, e2)
```

Comment:
>e1 [](start = 23, length = 2)

API nit, e1Id or something of the sort adds more context to the API parameter.

Reply:
In other APIs you use EntityId1, EntityId2.  Should probably be consistent here.

---
In reply to: [498409229](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/181#discussion_r498409229) [](ancestors = 498409229)